libraries/plugins/follow/follow_api.cpp

```diff
@@ -0,0 +1,358 @@
+#include <steemit/chain/account_object.hpp>
+
+#include <steemit/follow/follow_api.hpp>
+
+namespace golos {
+    namespace follow {
+
+        namespace detail {
+
+            inline void set_what(vector<follow_type> &what, uint16_t bitmask) {
+                if (bitmask & 1 << blog) {
+                    what.push_back(blog);
+                }
+                if (bitmask & 1 << ignore) {
+                    what.push_back(ignore);
+                }
+            }
+
+            class follow_api_impl {
+            public:
+                follow_api_impl(golos::app::application &_app)
```

Comment:
You said the other day that 'app' variable must be taken from appbase::app() rather then constructor parameter 
In fact u only using database from app so its beter to just add _db member 
_db(appbase::app().get_plugin<golos::plugins::chain::plugin>().db())